components/brave_extension/extension/brave_extension/_locales/en_US/messages.json

```diff
@@ -24,8 +24,12 @@
     "description": "Message telling the user to disable shields if a site appears broken"
   },
   "disabledMessage": {
-    "message": "You’re browsing this site without any privacy and security protections.",
-    "description": "Message telling the user that shields are disabled"
+    "message": "You’re browsing this site without Brave's privacy and security protections. Does it not work right with Shields up?",
```

Comment:
Super-nit: One of the apostrophe is slanted ("You're") but not the other one ("Brave's"). I imagine we have a standard for this though I don't know what it is.

Reply:
I guess the logical thing to do would be to copy the one that was already there... but definitely would be good to figure out/solidify the rationale here

Reply:
Just to be clear my "changes requested" review was for the HTTP URL, not for this :)

Reply:
haha, understood :) I'll make sure it's consistent by the time the URL is ready though

Reply:
All set, the slanted one is the correct one to use going forward.